Tasks/DownloadPackageV1/task.json

```diff
@@ -120,7 +120,7 @@
         {
             "target": "feed",
```

Comment:
version bump?

Reply:
218 is not deployed yet right?

Reply:
I thought it was needed - or the standared was - to bump the patch version - please correct me if I am wrong.

Reply:
I will bump it just to be safe